.eleventyignore

```diff
@@ -0,0 +1,8 @@
+Gemfile
+Gemfile.lock
+package.json
+LICENSE
+README.md
+node_modules
+scripts
+.github/
```

Comment:
Nit: Is the ending slash necessary?

Reply:
The slash just indicates it's a directory. It can be removed without issues

Reply:
removed